app/models/default_report_service.rb

```diff
@@ -0,0 +1,38 @@
+# This class mimics external_report.rb
+class DefaultReportService
+  DefaultReportServiceAppID   = 'DEFAULT_REPORT_SERVICE_CLIENT'
+  DefaultReportDomainMatchers = '*.concord.org concord-consortium.github.io localhost'
+  ReportViewUrl = 'https://concord-consortium.github.io/portal-report/'
+
+
+  ReportTokenValidFor = 2.hours
+
+  def self.instance
+    @instance || self.new()
+  end
+
+  def reportViewUrl
+    ENV['REPORT_VIEW_URL'] || ReportViewUrl
+  end
+
+  def initialize
+    found = Client.find_by_app_id(DefaultReportServiceAppID)
+    @client = found || Client.new({
+      app_id: DefaultReportServiceAppID,
+      name: DefaultReportServiceAppID,
+      app_secret: SecureRandom.uuid()
+    })
+    if @client.domain_matchers != DefaultReportDomainMatchers
+      @client.update_attribute(:domain_matchers, DefaultReportDomainMatchers)
+    end
```

Comment:
I think the DefaultReportDomainMatchers should also be pulled out of a ENV variable. Especially since this code updates the default report service with these domains.  By putting it in an ENV variable we can move the report, update the ENV, restart the app, and now the default report will be pointed at the new place.